src/components/content/content.stories.mdx

```diff
@@ -1,7 +1,7 @@
 import { Meta, Story, Preview, Props} from '@storybook/addon-docs/blocks';
-import Content from './content.js';
+import Content from './content.component.js';
 
-<Meta title="Content" parameters={{ info: { disable: true }}} />
+<Meta title="Design System/Content" parameters={{ info: { disable: true }}} />
```

Comment:
The Content component now appears in `Design System` and outside of the `Design System` in its own `Content` folder. Should we not have the test and the docs in one location so not to confuse consumers?